### PR TITLE
Rework logging functionality for main pipeline.

### PR DESCRIPTION
**Pull request by timstaley**
**Note change to pipeline.cfg**
This adds a line to the ``logging`` section of *pipeline.cfg*:

    [logging]
    colorlog = True

---------------

Partly in response to #501, but I've been meaning to tidy this up
for a while.

Note that we still lose all logs from pool-processes when
'multiprocessing' mode is employed. Something to work on.

### CODE REVIEW

File: tkp/distribute/serial/tasks.py [hunk 1]
```diff
@@ -7,16 +7,16 @@
 
 
 def persistence_node_step(images, image_cache_config, sigma, f):
-    logger.info("running persistence task")
```
Comment (gijzelaerr) [inline tkp/distribute/serial/tasks.py]: is this really something we want to change to debug level?

**Comment by gijzelaerr** [comment]
I never realised that we lose the logs from using multiprocessing?

**Comment by gijzelaerr** [comment]
apart from my questions I like this, looks much more readable to me. PR works, so almost good to go.

File: tkp/distribute/serial/tasks.py (hunk 1 above)
Comment (timstaley) [inline tkp/distribute/serial/tasks.py]: Well, we also note that we're starting this step from the main logic flow:
https://github.com/transientskp/tkp/blob/ee53bc4b92e35b309182fb6047a5a658ee834855/tkp/main.py#L95
so I figured this was quite redundant

**Comment by timstaley** [comment]
Yeah - try it - run a few test images in serial / multiprocessing mode and watch the difference. We lose all the logs from the multiproc tasks, i.e. blind-extraction mainly.

**Pull request merged**